2.Lets-Build-OTPublisher/Lets-Build-OTPublisher/ViewController.m

```diff
@@ -74,7 +74,7 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:
  */
 - (void)doConnect
 {
-    OTError *error;
+    OTError *error = nil;
```

Comment:
Yeah we've run into this in a few places -- if you don't initialize the declaration the variable ends up with whatever garbage came off the heap. I wonder if there's a compiler option or something that we're missing...